### DISCUSSION TITLE
Openssl 3.0.15 undefined symbol error

### DISCUSSION THREAD

**Discussion by Prabal1998** [openssl/openssl#25736] [Q&A]
1.Following error encountered during installation of Python (edited) that requires OpenSSL libraries.
![image](https://github.com/user-attachments/assets/ade4e492-94e6-4152-a012-e0ff720579c0)

2.Installed version
![image](https://github.com/user-attachments/assets/d82c5753-0f2a-4189-90f3-295455edab90)

Please help to resolve.

**Comment by tom-cosgrove-arm** [comment]
Did your errors occur while you were installing OpenSSL or while you were trying to build some software (e.g. Python) with a new version of OpenSSL? It would help if you would let us know what system you are on and what commands you were running.

If you're trying to build some software with OpenSSL 3.0.15 have you checked that the software is compatible with that version of OpenSSL?

And: questions are best asked as discussions - this doesn't seem to be an issue with OpenSSL itself

**Comment by Prabal1998** [comment]
Indeed, I was trying to build Python using OpenSSL.

**Comment by mattcaswell** [comment]
This question seems like something you should ask in a Python forum. Those warnings don't come from OpenSSL itself - they look like something that is failing when attempting to use OpenSSL.
